app/com/linkedin/drelephant/exceptions/azkaban/InsufficientLogException.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2016 LinkedIn Corp.
+ *
```

Comment:
Shouldn't it be better to have one exception class LogException and have different exception messages(InsufficientLog , LogNotFound) ? .